packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -705,14 +735,18 @@ detail: ${JSON.stringify(eventDetail)}`);
     });
     try {
       // send message
-      const chat: ChatMessage = await sendChatMessage(this.graph, this.chatId, content);
+      const chat: ChatMessage = await sendChatMessage(this.graph, this.chatId, updatedContent, mentions);
```

Comment:
perhaps we should be passing updatedContent as the content for the pending message to avoid the unstyled markup that shows briefly?

Reply:
This will also show <at>Display Name</at>. I have put a temporary message that displays the selected name then it flickers to show the updated mention in an mgt-people.